map-storage/src/MapStorageServer.ts

```diff
@@ -0,0 +1,86 @@
+import { sendUnaryData, ServerUnaryCall } from "@grpc/grpc-js";
+import { AreaType, ITiledMapRectangleObject } from "@workadventure/map-editor";
+import { mapsManager } from "./MapsManager";
+import {
+    EditMapCommandMessage,
+    EditMapCommandWithKeyMessage,
+    EmptyMessage,
+    MapStorageServer,
+    PingMessage,
+} from "./Messages/ts-proto-map-storage-generated/protos/messages";
+
+const mapStorageServer: MapStorageServer = {
+    ping(call: ServerUnaryCall<PingMessage, EmptyMessage>, callback: sendUnaryData<PingMessage>): void {
+        callback(null, call.request);
+    },
+    handleEditMapCommandWithKeyMessage(
+        call: ServerUnaryCall<EditMapCommandWithKeyMessage, EmptyMessage>,
+        callback: sendUnaryData<EditMapCommandMessage>
+    ): void {
+        const editMapCommandMessage = call.request.editMapCommandMessage;
+        if (!editMapCommandMessage || !editMapCommandMessage.editMapMessage?.message) {
+            callback(null, { id: "", editMapMessage: undefined });
+            return;
+        }
+        const gameMap = mapsManager.getGameMap(call.request.mapKey);
+        if (!gameMap) {
+            // TODO: Send an error?
+            callback(null, { id: editMapCommandMessage.id, editMapMessage: undefined });
+            return;
+        }
+        const editMapMessage = editMapCommandMessage.editMapMessage.message;
+        try {
+            switch (editMapMessage.$case) {
+                case "modifyAreaMessage": {
+                    const message = editMapMessage.modifyAreaMessage;
+                    const area = gameMap.getGameMapAreas().getArea(message.id, AreaType.Static);
+                    if (area) {
+                        const areaObjectConfig: ITiledMapRectangleObject = {
+                            ...area,
+                            ...message,
+                        };
+                        mapsManager.executeCommand(call.request.mapKey, {
+                            type: "UpdateAreaCommand",
+                            areaObjectConfig,
+                        });
+                    }
```

Comment:
Maybe we should console.log a message if the area is not found? (that can happen if someone deletes the area while someone else is updating it, but it should be rare enough so maybe we can log that?)